platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PcesWriterWiring.java

```diff
@@ -31,15 +32,15 @@
  * @param doneStreamingPcesInputWire        the input wire for the trigger to indicate that PCES streaming is complete
  * @param eventInputWire                    the input wire for events to be written
  * @param discontinuityInputWire            the input wire for PCES discontinuities
- * @param minimumGenerationNonAncientInput  the input wire for the minimum generation of non-ancient events
+ * @param nonAncientEventWindowInput  the input wire for the minimum generation of non-ancient events
  * @param minimumGenerationToStoreInputWire the input wire for the minimum generation of events to store
  * @param latestDurableSequenceNumberOutput the output wire for the latest durable sequence number
  */
 public record PcesWriterWiring(
         @NonNull InputWire<DoneStreamingPcesTrigger> doneStreamingPcesInputWire,
         @NonNull InputWire<GossipEvent> eventInputWire,
         @NonNull InputWire<Long> discontinuityInputWire,
-        @NonNull InputWire<Long> minimumGenerationNonAncientInput,
+        @NonNull InputWire<NonAncientEventWindow> nonAncientEventWindowInput,
         @NonNull InputWire<Long> minimumGenerationToStoreInputWire,
```

Comment:
member should be renamed

Reply:
Not sure I understand this comment. What should be renamed, and how?

Reply:
The bound method was renamed to `setMinimumAncientIdentifierToStore`, I would assume the name of the wire should change to `minimumAncientIdentifierToStoreInputWire`

Reply:
Ah, right. I was stuck looking at the line I changed and did not notice the line right beneath that. Renamed.